packages/editor/src/components/widgets/code-editor/CodeEditor.tsx

```diff
@@ -34,7 +34,7 @@ const CodeEditor = ({ value, onChange, context, macro, onMountFuncs, options, ..
         defaultValue={value}
         value={value}
         defaultLanguage={language}
-        defaultPath={`${language}/${contextPath}?location=${context.location}${context.type ? `&type=${context.type}` : ''}`}
+        defaultPath={`${language}/${contextPath}/${context.location}/${context.type ? context.type : ''}`}
```

Comment:
Only add / if we have a type as you do it in your test in the other PR?

Reply:
Makes no difference in the backend. But I've added a test for this on the engine